### ISSUE TITLE
[Documentation request] - Example with Next.js SSR

### ISSUE THREAD

**Issue by samithaf** [atlassian/react-sweet-state#169]
Thank you for this awesome library!! I used it on a large financial Application based on React Native and loved it. 

New React offical Docs encourage people to use Next.js or alternative as the starting point for production applications, It would be great if you can add some examples specially with SSR.

Thank you!

**Comment by BeginArjun** [comment]
Yep and with the new React 19 introducing Server Components and many other stuffs should we try to provide examples for those too?
